### PR TITLE
NOJIRA: Add "Apereo Welcoming" code of conduct

### PR DESCRIPTION
**Pull request by ChristianMurphy**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]
-   [x] documentation is changed or added

##### Description of change

Adds the "Apereo Welcoming" code of conduct to uPortal.
Alternative to #925.
Mailing list discussion <https://groups.google.com/a/apereo.org/d/msg/uportal-dev/hJas1FdYW1w/Zy71sMMMAgAJ>.

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+While at Apereo events or related social networking opportunities, speakers, presenters, and attendees should not engage in speech or actions that may be considered harassment or unwelcoming.
+
+The Apereo Foundation reserves the right take appropriate action, to warn an offender, to deny entrance to an offender, or to expel an offender without refund.
+
+We value your attendance at this event and want your experience to be positive and welcoming.
+
+Please report any concerns to the immediate attention:
+
+1. Duty Officer:  Name, contact info
+2. Duty Officer:  Name, contact
```
Comment (ChristianMurphy) [inline CODE_OF_CONDUCT.md]: Duty officers need names and contact info

**Comment by jimhelwig** [comment]
Instead of copying the policy, would it make sense to have a brief paragraph about uPortal following the Apereo policy and link to that?

**Comment by ChristianMurphy** [comment]
closing in favor of #931

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (apetro) [reply CODE_OF_CONDUCT.md]: While [this oversight was escalated to the Board in June 2017|https://groups.google.com/a/apereo.org/d/msg/uportal-dev/hJas1FdYW1w/xdiXrct0AQAJ], it is still not yet resolved.

Arguably uPortal should withdraw the `CODE_OF_CONDUCT.md` until this is resolved. We're not doing anyone any favors claiming to have a code of conduct that isn't actually operationalized and viable.

**Pull request closed without merging**